subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/reservation/client/kubernetes/Utils.kt

```diff
@@ -0,0 +1,3 @@
+package com.avito.instrumentation.reservation.client.kubernetes
+
+fun String.kubernetesName(): String = replace("_", "-").toLowerCase()
```

Comment:
Can it be at least internal?

Reply:
yes, thanks;
really struggling without lib mode; hope gradle 6.8 will be enough to force 1.4 kotlin in project

Reply:
Moved to `:kubernetes` module, which is implementation dependency for `instrumentation-tests`